Tasks/AzurePowerShellV4/InitializeAz.ps1

```diff
@@ -0,0 +1,46 @@
+[CmdletBinding()]
+param
+(
+    [String] [Parameter(Mandatory = $false)]
+    $targetAzurePs,
+
+    [String] [Parameter(Mandatory = $true)]
+    $serviceName,
+
+    [String] [Parameter(Mandatory = $true)]
+    $endpoint
```

Comment:
Mandatory input should come before optional imputs

Reply:
It is done